### ISSUE TITLE
Issues using forked alphafold for modifed colabfold 

### ISSUE THREAD

**Issue by michaelhla** [sokrypton/ColabFold#652]
## Expected Behavior
trying to test run a colabfold on a modified alphafold repo, should run batch.py on a test fasta file

## Current Behavior
alphafold.common is not recognized, modulenotimported error 

## Steps to Reproduce (for bugs)
follow the steps on contribute.md for local changes

## Context
Have tried updating the python path, importing manually with importlib

## Your Environment
running on linux machine, a100, not docker compatible

**Comment by michaelhla** [comment]
have also tried working directly in a colab notebook but Bio.SCOP not found when running `import colabfold.batch`

**Comment by milot-mirdita** [comment]
Are you using our alphafold fork: https://github.com/steineggerlab/alphafold

If not please replay your changes onto our fork. ColabFold is not meant to be run with DeepMind's AF2.

**Comment by michaelhla** [comment]
yeah just following instructions here, seems to be your af fork: https://github.com/sokrypton/ColabFold/blob/main/Contributing.md
